core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/ActiveDirectoryUserStoreManager.java

```diff
@@ -245,8 +245,8 @@ public void doUpdateCredential(String userName, Object newCredential, Object old
                 .getUserStoreProperty(LDAPConstants.USER_NAME_ATTRIBUTE);
         // String searchFilter =
         // realmConfig.getUserStoreProperty(LDAPConstants.USER_NAME_SEARCH_FILTER);
-        String searchFilter = "(&" + userListFilter + "(" + userNameAttribute + "=" + userName
-                + "))";
+        String searchFilter = "(&" + userListFilter + "(" + userNameAttribute + "=" +
```

Comment:
Is there any possibility that an ordinary use can exploit a setter in realmConfig to set "userListFilter" and "userNameAttribute" fields and then do a LDAP injection [1]?

[1] https://www.owasp.org/index.php/LDAP_injection